angular2-jwt.ts

```diff
@@ -56,57 +64,74 @@ export class AuthHttp {
 
   private _config: IAuthConfig;
   public tokenStream: Observable<string>;
+  private jwtHelper:JwtHelper;
 
   constructor(options: AuthConfig, private http: Http) {
     this._config = options.getConfig();
+    this.jwtHelper = new JwtHelper();
 
     this.tokenStream = new Observable((obs: any) => {
       obs.next(this._config.tokenGetter())
     });
   }
 
-  _request(url: string | Request, options?: RequestOptionsArgs) : Observable<Response> {
+  _request(req: Request) : Observable<Response> {
```

Comment:
I thought those were unused pieces - it looked like the only code flow to there was from the `get`, `post` etc... methods

If it's intended to satisfy the [Http](https://angular.io/docs/ts/latest/api/http/Http-class.html) class then shouldn't it be renamed to `request` instead of `_request`? I took the underscore to indicate private (probably having a Python flashback moment).